library/tinkerpop/pom.xml

```diff
@@ -27,6 +27,9 @@
 
     <properties>
         <tinkerpop.version>3.6.4</tinkerpop.version>
+
+        <!-- Don't run any of the standard tinkerpop tests by default-->
+        <test>!GafferPopGraphStructureStandardTest</test>
```

Comment:
It might be an option to run the tests but not fail on errors. If that can be done for a single test?

Reply:
The best could probably do is set the failure ignore property if the profile is activated [see](https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#testFailureIgnore)